src/easylink/pipeline_schema_constants/development.py

```diff
@@ -88,9 +88,30 @@
             )
         ],
         output_slots=[OutputSlot("step_3_main_output")],
+        nodes=[
+            BasicStep(
+                "step_3",
+                input_slots=[
+                    InputSlot(
+                        name="step_3_main_input",
+                        env_var="DUMMY_CONTAINER_MAIN_INPUT_FILE_PATHS",
+                        validator=validate_input_file_dummy,
+                    )
+                ],
+                output_slots=[OutputSlot("step_3_main_output")],
+            )
+        ],
+        edges=[
+            Edge(
+                in_node="step_3",
```

Comment:
I find `in_node` and `out_node` to be confusing names. I guess they mean where you go "in" to the edge from and then where you go "out" to, but I would more naturally read it as "in" is the node that this edge serves as an input to and "out" the node that the edge is the output from. 

At the risk of increasing verbosity, can we be more explicit with our naming here?

Reply:
sure, "source node" and "target node" would be consistent with Networkx.